server/fleet/datastore.go

```diff
@@ -511,6 +511,8 @@ type Datastore interface {
 
 	// ListOperationsSystems returns all operating systems (id, name, version)
 	ListOperatingSystems(ctx context.Context) ([]OperatingSystem, error)
+	// ListOperatingSystemsForPlatform returns all operating systems for the given platform
```

Comment:
Let's clarify the currently supported values for platform are `"darwin"` and `"windows"`.